src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Renamed.cs

```diff
@@ -150,7 +150,7 @@ public static void FileSystemWatcher_Moved_NestedDirectoryRootWithoutSubdirector
 
                 using (var dir2 = new TemporaryTestDirectory(Path.Combine(dir1.Path, "dir2")))
                 {
-                    Utility.ExpectNoEvent(createdOccured, "dir2 created", Utility.WaitForCreationTimeoutInMs);
+                    Utility.ExpectNoEvent(createdOccured, "dir2 created");
```

Comment:
Cool :smile: And I completely agree: as someone who spends a lot of time fighting failing tests on github, non-deterministically failing tests really get on my nerves. In this case, I don't love it, but it just means the test will pass even though it maybe should have failed. In general we have a fair amount of that in these FSW tests, as there are a bunch of tests that try to verify the absence of something happening at some point in the future, and so at some point we have to say "well, if it hasn't happened yet, it probably won't" and declare success, even though it could still happen.